Misc/NEWS.d/next/Windows/2020-06-19-00-14-52.bpo-40851.0-3EJP.rst

```diff
@@ -0,0 +1,2 @@
+On Windows, :class:`subprocess.Popen` no longer sets ``wShowWindow`` if the
+``startupinfo`` argument has the flag ``STARTF_USESHOWWINDOW``.
```

Comment:
I think this should clarify that it's limited to "when `shell=True` is passed". 

Also, the documentation of [`STARTUPINFO.wShowWindow`](https://docs.python.org/3/library/subprocess.html#subprocess.STARTUPINFO.wShowWindow) needs to be updated. For example: "SW_HIDE is provided for this attribute. It is used when Popen is called with `shell=True`, unless `dwFlags` includes `STARTF_USESHOWWINDOW`".